src/components/layout/Footer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { FiInstagram, FiFacebook, FiYoutube } from 'react-icons/fi'
+import { Button } from '../ui/buttons/button'
```

Comment:
Organicemos los imports 
```suggestion
import React from 'react'

// Components
import { FiInstagram, FiFacebook, FiYoutube } from 'react-icons/fi'
import { Button } from '../ui/buttons/button'
```

Reply:
entendido 11chuchu11